libs/community/tests/integration_tests/document_loaders/docker-compose/postgresql.yml

```diff
@@ -0,0 +1,19 @@
+version: "3"
+
+services:
+  postgresql:
+    image: postgres:16
+    environment:
+      - POSTGRES_HOST_AUTH_METHOD=trust
+    ports:
+      - "5432:5432"
```

Comment:
If you have any cycles, It'll be good to some password here and use a non standard port; I'm worried that at some point some poor user will lose their existing dev database due to a coincidence

Reply:
added password